RELEASE-NOTES.md

```diff
@@ -20,6 +21,9 @@
 
 ### Maintenance
 - Tuning results no longer leak into sequentially sampled `Metropolis` chains (see #3733 and #3796).
+- Deprecated `sd` in version 3.7 has been replaced by `sigma` now raises `DeprecationWarning` on using `sd` in continuous, mixed and timeseries distributions. (see #3837 and #3688).
```

Comment:
Is the beginning of the sentence missing some words or punctuation?

Reply:
this line was actually a duplicate. I changed the wording in the deprecations section